docker/README.md

```diff
@@ -0,0 +1,61 @@
+# VRX Docker support
+
+# Overview
+
+The directory contains a series of Dockerfiles designed to serve different setups
+for VRX. The usage is mainly controlled by `docker-compose` although docker files
+can be used independently if desired:
+
+ * `Dockerfile.base`: built on top of osrf desktop-full. Install
+   gz-harmonic from packages.osrfoundation.org repostiory and
+   the VRX dependencies using rosdep.
+
+   * `Dockerfile.builder`: built on top of `vrx-base`. Compiles the vrx packages
```

Comment:
I think we should explain what is `vrx-base`.

Reply:
Include the image names in [ad84a95](https://github.com/osrf/vrx/pull/817/commits/ad84a9577933782ac2529a3104cc5b2bc14806e2)